anyrun/src/main.rs

```diff
@@ -506,27 +516,29 @@ fn handle_matches(
             .name(style_names::MATCH)
             .hexpand(true)
             .build();
-        if let ROption::RSome(icon) = &_match.icon {
-            let mut builder = gtk::Image::builder()
-                .name(style_names::MATCH)
-                .pixel_size(32);
-
-            let path = PathBuf::from(icon.as_str());
-
-            // If the icon path is absolute, load that file
-            if path.is_absolute() {
-                match gdk_pixbuf::Pixbuf::from_file_at_size(icon.as_str(), 32, 32) {
-                    Ok(pixbuf) => builder = builder.pixbuf(&pixbuf),
-                    Err(why) => {
-                        println!("Failed to load icon file: {}", why);
-                        builder = builder.icon_name("image-missing"); // Set "broken" icon
+        if !hide_icons {
```

Comment:
This if statement could be collapsed to be just `if let ROption::RSome(icon) = &_match.icon && !hide_icons.

Reply:
if i do that rust_analyzer complains with "let expressions in this position are unstable"